docs/install/knative-with-operators.md

```diff
@@ -25,10 +27,55 @@ Disk storage to at least 30 GB. If you have multiple nodes for your cluster, set
 
 ## Limitations of Knative Operators:
 
-Knative Operators use custom resources (CRs) to configure your Knative deployment.
+Knative Operators are still in Alpha phase. They have not been tested in a production environment, and should be used
+for development or test purposes only.
 
- - Currently, the CRs included with Knative Operators do not provide high availability (HA) capabilities.
- - Knative Operators have not been tested in a production environment, and should be used for development or test purposes only.
+## Install Knative with All-In-One operator
+
+Knative has released an All-In-One operator to install both Serving and Eventing components. You can find the information
+about the releases on the [Releases page](https://github.com/knative-sandbox/operator/releases).
+
+### Installing the All-In-One Knative Operator
+
+__From releases__:
+
+Install the latest Knative operator with the following command:
+
+```
+kubectl apply -f https://github.com/knative-sandbox/operator/releases/latest/download/operator.yaml
+```
+
+__From source code__:
+
+You can also install Knative Operators from source using `ko`.
+
+1. Install the [ko](https://github.com/google/ko) build tool.
+1. Download the source code using the following command:
+
+```
+git clone https://github.com/knative-sandbox/operator.git
+```
+
+1. Install the operator in the root directory of the source using the following command:
+
+```
+ko apply -f config/
+```
+
+The All-In-One operator installs both Serving and Eventing operators. Go through the following sections to verify your operator:
+
+- [Verify Knative Serving Operator](#verify-the-operator-installation)
+- [Verify Knative Eventing Operator](#verify-the-operator-installation-1)
+
+Install Knative Serving and Eventing with Operator CRs:
+
+- [Install Knative Serving](#installing-the-knative-serving-component)
+- [Install Knative Eventing](#installing-the-knative-eventing-component)
+
+If you would like to install Knative Serving and Knative Eventing separately, go through the following sections:
+
+- [Install Knative Serving using Operator](#installing-the-knative-serving-operator)
+- [Install Knative Eventing using Operator](#installing-the-knative-serving-operator)
```

Comment:
I think you need a separate header here (or above, maybe near line 26 after the prerequisites are fulfilled) for these links. Right now, this reads as if you need to do these two steps at the end of installing the unified operator.

Reply:
I added a header before these two links.